### PR TITLE
PLANET-7628: Remove Articles block

### PR DESCRIPTION
**Pull request by mardelnet**
Ref: https://jira.greenpeace.org/browse/PLANET-7628

<hr>

**DESCRIPTION:** 
This PR removes the Articles block code from the P4 plugin.

<hr>

**RELATED PR:**
https://github.com/greenpeace/planet4-master-theme/pull/2405

### CODE REVIEW

**Comment by planet-4** [comment]
### Test instance is ready :rocket:

:new_moon: [pluto](https://www-dev.greenpeace.org/test-pluto) | [admin](https://www-dev.greenpeace.org/test-pluto/wp-admin/) | [blocks report](https://www-dev.greenpeace.org/test-pluto/wp-admin/?page=plugin_blocks_report) | [CircleCI](https://app.circleci.com/pipelines/github/greenpeace/planet4-test-pluto) | [composer-local.json](https://github.com/greenpeace/planet4-test-pluto/blob/main/composer-local.json)

:watch: 2024.10.14 09:17:42

**Review by mleray — COMMENTED** [review]
I think we should also move [this e2e test](https://github.com/greenpeace/planet4-plugin-gutenberg-blocks/blob/main/tests/e2e/related-articles.spec.js) to the theme

**Comment by mardelnet** [comment]
> I think we should also move [this e2e test](https://github.com/greenpeace/planet4-plugin-gutenberg-blocks/blob/main/tests/e2e/related-articles.spec.js) to the theme

Good catch! Done.

File: assets/src/blocks/Articles/ArticlesEditor.js [hunk 1]
```diff
@@ -1,162 +0,0 @@
-import {
-  CheckboxControl,
-  TextControl as BaseTextControl,
-  PanelBody,
-  Tooltip,
-} from '@wordpress/components';
-import {InspectorControls, RichText} from '@wordpress/block-editor';
-import withCharacterCounter from '../../components/withCharacterCounter/withCharacterCounter';
-import TagSelector from '../../components/TagSelector/TagSelector';
-import {PostSelector} from '../../components/PostSelector/PostSelector';
```
Comment (mleray) [inline assets/src/blocks/Articles/ArticlesEditor.js:10]: If I'm not mistaken, this component isn't used anywhere else so it can be removed too?

Comment (mardelnet) [reply assets/src/blocks/Articles/ArticlesEditor.js:10]: Done!

**Review by mleray — APPROVED** [review]
🧹 👍

**Pull request merged**